GDJS/GDJS/IDE/ExporterHelper.cpp

```diff
@@ -191,6 +192,17 @@ bool ExporterHelper::ExportProjectForPixiPreview(
     previousTime = LogTimeSpent("Events code export", previousTime);
   }
 
+  auto projectUsedResources =
+      gd::UsedResourcesFinder::FindProjectUsedResources(exportedProject);
+  std::vector<std::set<gd::String>> layersUsedResources;
+  for (std::size_t layoutIndex = 0;
+       layoutIndex < exportedProject.GetLayoutsCount(); layoutIndex++) {
+    auto &layout = exportedProject.GetLayout(layoutIndex);
```

Comment:
Nit: Can this be (and in the other file)
```suggestion
    const auto &layout = exportedProject.GetLayout(layoutIndex);
```
?

Reply:
Sadly, it's not possible because there is no constant version of the resource exposer.